apache2/modsecurity.c

```diff
@@ -122,6 +122,37 @@ msc_engine *modsecurity_create(apr_pool_t *mp, int processing_mode) {
     return msce;
 }
 
+int acquire_global_lock(apr_global_mutex_t *lock, apr_pool_t *mp) {
+    apr_status_t rc;
+    apr_file_t *lock_name;
+    const char *temp_dir;
+    const char *filename;
+
+    // get platform temp dir
+    rc = apr_temp_dir_get(&temp_dir, mp);
+    if (rc != APR_SUCCESS) {
+        ap_log_perror("ModSecurity: Could not get temp dir");
```

Comment:
Based on the [reference](https://nightlies.apache.org/httpd/trunk/doxygen/group__APACHE__CORE__LOG.html#ga2c7707a1e1930f5b5a2889ae66243aba) of `ap_log_perror()` it seems that few arguments are missing - see the correct syntax below in line [143](https://github.com/owasp-modsecurity/ModSecurity/pull/3149/files#diff-389d3c26e7333c1d93edeb63987800af15b4fe765e4a5a3c7455e13ca21a49c0R143).

I didn't check the cause of [failed CI tests](https://github.com/owasp-modsecurity/ModSecurity/pull/3149/checks), but I'm almost sure the reason is this.

Reply:
Agreed. Did it for the others, but oversaw this one. Changed.